sourcemash/frontend/templates/splash.tpl

```diff
@@ -0,0 +1,6 @@
+<div class="row">
+  <div class="col s12 valign-wrapper">
+    <img src="/static/solologo.svg">
+    <img src="/static/logotext.svg">
+  </div>
+</div>
```

Comment:
Just thought this would be a good way to show off the logo since the splash template was empty anyway. You can delete it if you want, but just added it so we don't have to stare at a blank white index page every time we hit sourcemash.com.